assets/templates/passphrase_renew.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <title>{{.Title}}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="{{asset .Domain "/fonts/fonts.css"}}">
```

Comment:
Can you add `.ContextName`?

Reply:
oh for this too? I wonder what it is for because some other templates like `login.html` don't have it on `fonts.css` so I thought it wasn't needed.

Reply:
It's better to put it on every call to `asset` in templates (but it's not a big deal if it is missing). For the moment, we don't override `fonts.css` via the dynamic asset, but if we want to do that one day, it will be nice to have it ready.

Reply:
Copy that 👍